Overview.src.html

```diff
@@ -160,7 +160,6 @@ <h2 class="short">Conformance</h2>
  <li><dfn>process request body</dfn>
  <li><dfn>process request end-of-file</dfn>
  <li><dfn>process response</dfn>
- <li><dfn>process response body</dfn>
```

Comment:
I suspect the idea is to revamp XMLHttpRequest to use streams directly rather than this indirection.